source/Preferences.cpp

```diff
@@ -41,7 +41,8 @@ namespace {
 void Preferences::Load()
 {
 	// These settings should be on by default. There is no need to specify
-	// values for settings that are off by default.
+	// values for settings that are off by default unless they are not
+	// shown in PreferencesPanel.
```

Comment:
I don't see any reason not to add this to the preferences panel, since there's plenty of space (in the "Other" section).

Reply:
Should those be alphabetized?